identity/identity.go

```diff
@@ -31,6 +31,52 @@ var IDService Service
 
 type CentID [CentIDLength]byte
 
+// IdentityConfig holds information about the identity
+type IdentityConfig struct {
+	ID   CentID
+	Keys map[int]IdentityKey
+}
+
+// IdentityKey represents a key pair
+type IdentityKey struct {
+	PublicKey  []byte
+	PrivateKey []byte
+}
+
+// GetIdentityConfig returns the identity and keys associated with the node
+func GetIdentityConfig() (*IdentityConfig, error) {
```

Comment:
I think we `GetIdentityConfig` can take a purpose like `KeyPurposeSigning` return that config alone. What do you think?

Reply:
I thought about that at the beginning, but sounds that reading 1 or 3 keys does not create a significant penalty. 
Knowing that in the future we might read it once and pass it to through the context.